rmgpy/molecule/molecule.py

```diff
@@ -402,6 +401,30 @@ def setSpinMultiplicity(self,spinMultiplicity):
         if self.spinMultiplicity < 0:
             raise ActionError('Unable to update Atom due to spin multiplicity : Invalid spin multiplicity set "{0}".'.format(self.spinMultiplicity))
         self.updateCharge()
+
+    def getBondOrdersForAtom(self):
+        """
+        This helper function is to help calculate total bond orders for an
+        input atom.
+
+        Some special consideration for the order `B` bond. For atoms having 
+        three `B` bonds, the order for each is 4/3.0, while for atoms having other
+        than three `B` bonds, the order for  each is 3/2.0
+        """
+        num_B_bond = 0
+        order = 0
+        for _, bond in self.bonds.items():
```

Comment:
thanks for pointing it out, I was trying to use `self.bonds.iteritems()`instead of `self.bonds.items()`, since former is a type of `iterator/generator` which is more memory efficient. For more detail, you can take a look [here](http://stackoverflow.com/questions/24189379/what-is-the-advantage-of-iteritems).